internal/kafka/internal/services/kafka.go

```diff
@@ -905,6 +908,8 @@ func (k *kafkaService) VerifyAndUpdateKafkaAdmin(ctx context.Context, kafkaReque
 		return errors.NewWithCause(errors.ErrorGeneral, err, "Failed to update kafka")
 	}
 
+	metrics.UpdateKafkaRequestsStatusSinceLastSetMetric(constants2.KafkaStatus(kafkaRequest.Status), kafkaRequest.ID, kafkaRequest.ClusterID)
```

Comment:
Is there a way for us to check the current status of the Kafka instance here? We probably only want to update this metric if the status is updated to a different state.

Reply:
I could add additional boolean param to indicate that status will be updated. would it make sense? unfortunately the metric can't be exposed from within admin kafka handler, because we can't be sure that the update will be successful

Reply:
I realized that I can actually do it in the handler. will push shortly with all of the updates